strictdoc/export/html/generators/source_file_view_generator.py

```diff
@@ -154,7 +160,9 @@ def get_pygmented_source_lines(
         pygmented_source_file_content = pygmented_source_file_content[
             slice_start:slice_end
         ]
-        pygmented_source_file_lines = pygmented_source_file_content.split("\n")
+        pygmented_source_file_lines: List[Union[str, RangeMarkerTuple]] = list(
```

Comment:
This is for mypy... Honestly I don't understand why we have to make a list()-copy of `pygmented_source_file_content.split("\n")`. Without list(), mypy would claim
```
error: Incompatible types in assignment (expression has type "List[str]", variable has type "List[Union[str, RangeMarkerTuple]]")
```

I know assignment would be a problem without explicit annotation of `pygmented_source_file_lines` because of list invariance. But the annotation is there... why do we still need list()?

Reply:
I added this to the Code climate ticket. I will review this at some point and see if I can resolve it in a different way. For now, low priority. 

https://github.com/strictdoc-project/strictdoc/issues/1912